packages/contracts/script/utils/exporter.js

```diff
@@ -3,19 +3,31 @@ require('dotenv').config();
 const path = require('path');
 const fs = require('fs');
 const { createClient } = require('@supabase/supabase-js');
+const { load } = require('js-toml');
+
+// Loads the environment-specified TOML configuration file.
+async function loadConfiguration() {
+  const env = process.env.ENVIRONMENT;
+  const configFile = path.resolve(__dirname, `../../resource/${env}.toml`);
+  console.log(`Loading configuration from: '${configFile}'`);
+  const toml = fs.readFileSync(configFile, 'utf8');
+  return load(toml);
+}
```

Comment:
split out this logic so it can be shared across these scripts.  similar will also be needed in the other packages.